test/format/ops.c

```diff
@@ -992,6 +993,13 @@ ops(void *arg)
     truncate_op = mmrand(&tinfo->data_rnd, 100, 10 * WT_THOUSAND);
 
     for (intxn = false; !tinfo->quit;) {
+        if (GV(OPS_THROTTLE)) {
+            /* Sleep first to avoid the burst when all threads start. */
+            /* Calculate max delay so that per-table ops/sec is as set. */
+            u_int throttle_delay = 2 * GV(OPS_THROTTLE_SLEEP_US) * GV(RUNS_THREADS) / (ntables > 0 ? ntables : 1);
```

Comment:
I think I'd put this assignment outside the loop.  Maybe rename as `max_throttle_delay`?  And please improve the comment so we know why it is multiplying by 2.

Reply:
Done